src/uu/ps/src/picker.rs

```diff
@@ -54,13 +53,14 @@ fn time(proc_info: RefCell<ProcessInformation>) -> String {
     let cumulative_cpu_time = {
         let utime = proc_info.borrow_mut().stat()[13].parse::<i64>().unwrap();
         let stime = proc_info.borrow_mut().stat()[14].parse::<i64>().unwrap();
-        utime + stime
+        (utime + stime) / 100
     };
 
-    DateTime::from_timestamp_millis(cumulative_cpu_time)
-        .unwrap()
-        .format("%H:%M:%S")
-        .to_string()
+    let hours = &cumulative_cpu_time / 3600;
+    let minutes = (&cumulative_cpu_time % 3600) / 60;
+    let seconds = &cumulative_cpu_time % 60;
+
+    format!("{:02}:{:02}:{:02}", hours, minutes, seconds)
```

Comment:
Why use custom formatting here?

Reply:
According to the manpage, the longest unit of `TIME` is day, if the time is longer than a month, `DateTime` will output wrongly.